app/src/main/resources/config/application.yml

```diff
@@ -140,5 +140,18 @@ nl-portal:
                   allowedHeaders: "*"
                   allowedOrigins: "*"
                   allowedMethods: "*"
-        #endpoints:
-            #unsecured: /playground, /actuator/**
+        endpoints:
+            unsecured: /api/payment/**
+    payment:
+        ogone:
+            url: https://secure.ogone.com/ncol/test/orderstandard.asp
+            sha-out-parameters: AAVADDRESS,AAVCHECK,AAVMAIL,AAVNAME,AAVPHONE,AAVZIP,ACCEPTANCE,ALIAS,AMOUNT,BIC,BIN,BRAND,CARDNO,CCCTY,CN,COLLECTOR_BIC,COLLECTOR_IBAN,COMPLUS,CREATION_STATUS,CREDITDEBIT,CURRENCY,CVCCHECK,DCC_COMMPERCENTAGE,DCC_CONVAMOUNT,DCC_CONVCCY,DCC_EXCHRATE,DCC_EXCHRATESOURCE,DCC_EXCHRATETS,DCC_INDICATOR,DCC_MARGINPERCENTAGE,DCC_VALIDHOURS,DIGESTCARDNO,ECI,ED,EMAIL,ENCCARDNO,FXAMOUNT,FXCURRENCY,IP,IPCTY,MANDATEID,MOBILEMODE,NBREMAILUSAGE,NBRIPUSAGE,NBRIPUSAGE_ALLTX,NBRUSAGE,NCERROR,ORDERID,PAYID,PAYIDSUB,PAYMENT_REFERENCE,PM,SCO_CATEGORY,SCORING,SEQUENCETYPE,SIGNDATE,STATUS,SUBBRAND,SUBSCRIPTION_ID,TRXDATE,VC
+            configurations:
+                belastingzaken:
+                    pspId: TAX
```

Comment:
Is de combinatie van pspId en shaInKey en shaOutKey niet een secret voor test omgeving? of zijn dit dummies?

Reply:
de pspId, shaInKey en shaOutKey zijn pure dummies. Op TST moeten via de ENV de correcte ingeladen worden